### PR TITLE
MDSAAS-202 Upgrade to Node.js 10.16.3 LTS runtime

### PR DESCRIPTION
**Pull request by drtyh2o**
## PR Checklist

 - [ ] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
 - [ ] briefly describe the changes in this PR
 - [ ] mark as draft if should not be merged
 - [ ] write tests for all new functionality

## Impacts
- [ ] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance
- [ ] Daily
- [ ] Native
- [ ] Policy Author

### CODE REVIEW

**Comment by macsj200** [comment]
🎉 This is exciting!

Are there any gotchas we should look out for?

**Comment by wesleyfsmith** [comment]
I'm also super excited about this! Especially since node 8 is getting the end of life treatment soon I think. Are there any considerations around our other tooling, namely yarn and typescript compilation if we are switching to a new node version?

**Comment by drtyh2o** [comment]
> I'm also super excited about this! Especially since node 8 is getting the end of life treatment soon I think. Are there any considerations around our other tooling, namely yarn and typescript compilation if we are switching to a new node version?

We were stuck on Node.js 8.10 because the `DeprecationWarning` messages that are generated in Node.js 10 could not be suppressed and were spamming the logs. Now that we control the node entry point, we can run with the `--no-deprecation` flag. I have been doing some testing locally and all of the tooling seems to be working fine.

**Comment by drtyh2o** [comment]
The build will fail until `CodeShip` is updated to the Node.js 10.16.3 runtime. We will update CodeShip once we are ready for this PR to merge.

File: container-images/mds-agency/Dockerfile [hunk 1]
```diff
@@ -8,4 +8,4 @@ COPY dist/* /mds/
 
 WORKDIR /mds
 
-ENTRYPOINT ["/sbin/tini", "node", "index.js"]
+ENTRYPOINT ["/sbin/tini", "node", "--no-deprecation", "index.js"]
```
Comment (invertigo) [inline container-images/mds-agency/Dockerfile]: 🥇

**Comment by invertigo** [comment]
Looks good!  This reminds me that we can consolidate down to a single Dockerfile for better DRY-compliance.  in `yarn image` you can specify `docker build -f ../Dockerfile ...` to override the default.

**Pull request merged**